zillow_crawler/main.py

```diff
@@ -59,6 +64,16 @@ def main():
         plt.bar(cities, prices, width=1)
         plt.show()
 
+        renderables = [
+            Panel(
+                f"[bold]{listing.address}[/bold]\n\nPrice: [green]${listing.price:0.2f}[/green]"
+                f"\nSqft: {listing.sqft}\nBedrooms: {listing.bedrooms}\nBathrooms: {listing.bathrooms}",
+                border_style="yellow",
+            )
+            for listing in listings            
+        ]
+        live.update(Panel.fit(Columns(renderables, equal=True), title = "Houses Located"))
+
         # renderables = [
         #     Panel(
         #         f"[bold italic]{book.book_name}[/bold italic]\n[green]{book.product_price}[/green]",
```

Comment:
We can probably eliminate the comment code if we do not use it.  Keep the comments that explain the code though.

Reply:
done